### PR TITLE
First commit: answer the self questions in README file.

### PR DESCRIPTION
**Pull request by moss-sc-moss**
@wcolts2000 these are the answers to the Questions in the README FIle.

### CODE REVIEW

**Comment by moss-sc-moss** [comment]
just because im nervous and did  a lot of commits and nothing bad happened

File: README.md [hunk 1]
```diff
@@ -34,14 +34,27 @@ Edit this document to include your answers after each question. Make sure to lea
...
+#### It is a way to customize your CSS in a way that makes both enterprise level designs all the way down to SPA easier to design and not get caught up in syntactic errors or verbose mark up. its like customizeable key shortcuts but for your CSS. And it looks more like actual code than CSS syntax.
+
 5. What is your favorite concept in preprocessing? What is the concept that gives you the most trouble?
 
+#### I enjoy the variables and being able to go about my color palattes in a way where I have control of the name and not subjected to unknown hex decinmal numbers when you are working front end you have ot be on the same page as UI/UX Designers. Mixins once I finally learned how to do them properly cut out some verbose syntax and allows for both a clean read and concise but manageable code. I am always concerned with nesting and specificity.
+
+
 You are expected to be able to answer all these questions. Your responses contribute to your Sprint Challenge grade. Skipping this section *will* prevent you from passing this challenge.
 
 ## Project Set Up
```
Comment (wcolts2000) [inline README.md:60]: very thorough answers. i like it!

File: index.html [hunk 2]
```diff
@@ -16,48 +19,69 @@
...
+    </header>
+  </div>
   <div class="container home">
 
     <header class="intro">
-       <!-- image code here -->
-
+      <!-- image code here DONT FORGET TO USE ATTRIBUTE ALT="describe img" FOR ACCESSIBILITY -->
+      <img src="img/main-header.png" alt="main header image of earth with a gutter added and image centered">
       <h2>Lift Off</h2>
```
Comment (wcolts2000) [inline index.html:44]: in alt tags you will want to describe what the picture is of, not its positioning and styling on the page. a visually impaired user wont know how to reference what a gutter and centered image is in relation to the rest of the page

File: less/home-page.less [hunk 3]
```diff
@@ -19,11 +23,25 @@
...
       width: 28%;
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
+
+      @media @phone {
+        width: 90%;
+      }
+
+      &.sky, &.launch {
```
Comment (wcolts2000) [inline less/home-page.less:40]: good job with nesting and media queries and utilizing the variables you have

File: less/home-page.less [hunk 4]
```diff
@@ -46,9 +64,19 @@
     display: flex;
     justify-content: center;
 
+    @media @phone {
+      flex-direction: column-reverse;
+      align-items: center;
```
Comment (wcolts2000) [inline less/home-page.less:69]: good catch using column reverse here to get it to position the way you want

File: less/home-page.less [hunk 5]
```diff
@@ -58,12 +86,27 @@
       }
 
       .btn {
+        display: inline-block;
+        .customButton(@btn-bg-color, @porcelain);
+        font-size: 2rem;
+        padding: 1rem 2rem;
```
Comment (wcolts2000) [inline less/home-page.less:92]: nice utilization of parametric mixins here

File: less/mixins.less [hunk 6]
```diff
@@ -1 +1,21 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.centeredColumn() {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (wcolts2000) [inline less/mixins.less:6]: i love using mixins for flex properties, good job here

File: less/navigation.less [hunk 7]
```diff
@@ -1 +1,300 @@
...
+
+        @media (max-width: 12rem * 4.4rem) {
+            width: 100%;
+        }
+
+        a {
+            font-size: 1.8rem;
+        }
+    }
+}
```
Comment (wcolts2000) [inline less/navigation.less:300]: i would remove all the commented dead code here

**Review by wcolts2000 — APPROVED** [review]

**Pull request merged**